src/apps/api/serializers/submissions.py

```diff
@@ -105,24 +110,32 @@ class Meta:
     def get_filename(self, instance):
         return basename(instance.data.data_file.name)
 
-    # TODO: Validate the user is a participant in this competition.phase
-
     def create(self, validated_data):
-        validated_data["owner"] = self.context['owner']
         sub = super().create(validated_data)
         sub.start()
         return sub
 
     def validate(self, attrs):
         data = super().validate(attrs)
+
+        if not self.instance:
```

Comment:
why does this check matter?

Reply:
add a comment here explaining that `self.instance` only exists in an update.

Reply:
added